homepage/homepage/start.sh

```diff
@@ -48,6 +48,9 @@ if [[ -n "${HOMEPAGE_TEMPLATE_REPO}" ]]; then
                 cat "${file}" | envsubst > $out_path
                 echo "Rendered template file: $out_path"
             done
+            rm -rf ${TMP_CLONE}/.* && rm ${TMP_CLONE}/Dockerfile && rm ${TMP_CLONE}/*.yaml
```

Comment:
if the file isnt there, rm will fail without `-f`

Why would Dockerfile be in the repo?

Reply:
can't force remove .* either because otherwise:

````
homepage-homepage-1  | rm: can't remove '.' or '..'
homepage-homepage-1  | rm: can't remove '.' or '..'
````